posthog/models/team/team.py

```diff
@@ -293,6 +309,27 @@ def aggregate_users_by_distinct_id(self) -> bool:
 
     objects: TeamManager = TeamManager()
 
+    def validate_payload(self, message: str, hash: str):
+        """
+        Based on hmac validation
+        """
+        # TODO: Do I need to consider any time-based validation here?
```

Comment:
Ideally yeah although its definitely harder for the implementer. Would be awesome for very sensitive operations to include a `?t=NOW` param as part of the message that can then be validated both in terms of time and content. That can always be added later though when we have a _really_ high need for validation.